platforms/chibios/converters/promicro_to_stemcell/_pin_defs.h

```diff
@@ -46,12 +46,6 @@
 #define B2 PAL_LINE(GPIOA, 7)
 #define B6 PAL_LINE(GPIOA, 4)
 
-// Extra elite-c compatible pinout
-#define B7 PAL_LINE(GPIOC, 13)
-#define D5 PAL_LINE(GPIOC, 14)
-#define C7 PAL_LINE(GPIOC, 15)
-#define F1 PAL_LINE(GPIOA, 0)
-#define F0 PAL_LINE(GPIOA, 1)
-
-// TX/RX pins of promicro
-#define B0 PAL_LINE(GPIOA, 9) // unconnected pin
+// LEDs
+#define D5 PAL_LINE(GPIOC, 14) // bottom row
+#define B0 PAL_LINE(GPIOA, 9)  // unconnected pin
```

Comment:
These are accessible LEDs on a promicro, so if the replacement implements them should there be a mapping?

Reply:
We dont really have a decent way to stub out any gpio access so mapping to rando pins is the common practice. That way any LED use in the pre-converted code does not error out.

However in this particular case, it seems the stemcell has a single LED.